substratevm/mx.substratevm/suite.py

```diff
@@ -1454,6 +1483,7 @@
             "dependencies": [
                 "com.oracle.svm.util",
                 "com.oracle.graal.pointsto",
+                "com.oracle.graal.pointsto.standalone",
```

Comment:
We do not want the new `standalone` project in any existing distribution. At least while it is under development, we do not want to ship it in GraalVM releases.

Reply:
Put it to its own jar file named standalone-pointsto.jar